taskcluster/test/test_default_params.py

```diff
@@ -0,0 +1,97 @@
+                            "taskId": "build-docker-image-base",
+                        },
+                        {
+                            "namespace": "translations.cache.level-3.docker-images.v2.test.hash.{digest_test}",
+                            "taskId": "build-docker-image-test",
+                        },
+                        {
+                            "namespace": "translations.cache.level-3.docker-images.v2.toolchain-build.hash.{digest_toolchain}",
+                            "taskId": "build-docker-image-toolchain-build",
+                        },
+                        {
+                            "namespace": "translations.cache.level-3.docker-images.v2.train.hash.{digest_train}",
+                            "taskId": "build-docker-image-train",
+                        },
+                    ],
+                },
+                "status_code": 200,
+            },
+        ],
+    },
+    {
+        "method": "POST",
+        "url": "https://firefox-ci-tc.services.mozilla.com/api/queue/v1/tasks/status",
+        "responses": [
+            {
+                "json": {
+                    "statuses": [
+                        {
+                            "status": {
+                                "state": "completed",
+                                "expires": "3024-08-21T22:37:28.781Z",
+                            },
+                            "taskId": "build-docker-image-base",
+                        },
+                        {
+                            "status": {
+                                "state": "completed",
+                                "expires": "3024-08-21T22:37:28.781Z",
+                            },
+                            "taskId": "build-docker-image-test",
+                        },
+                        {
+                            "status": {
+                                "state": "completed",
+                                "expires": "3024-08-21T22:37:28.781Z",
+                            },
+                            "taskId": "build-docker-image-toolchain-build",
+                        },
+                        {
+                            "status": {
+                                "state": "completed",
+                                "expires": "3024-08-21T22:37:28.781Z",
+                            },
+                            "taskId": "build-docker-image-train",
+                        },
+                    ],
+                },
+                "status_code": 200,
+            },
+        ],
+    },
+]
+
+
+def test_last_task_is_targeted(target_task_set):
+    """Ensure that the last task in the pipeline is targeted by default"""
+    assert any([task == "all-ru-en-1" for task in target_task_set.tasks])
+
+
+def test_cached_tasks_optimized_away(optimized_task_graph):
+    """Ensure that any tasks found in a cache route are _not_ present
```

Comment:
Thought: It's hard to tell what the types of the tests are with the fixture magic. Again, I think this is standard practice with pytest, but it's not great for discoverability.

Suggestion (optional): Adding at a minimum type hints here would be helpful.

Reply:
Will do.